README.md

```diff
@@ -21,6 +21,44 @@ You can find the latest Spark documentation, including a programming
 guide, on the [project web page](https://spark.apache.org/documentation.html).
 This README file only contains basic setup instructions.
 
+# Daily Build Pipeline Status
+
+| Branch     | Topic                                                  | Status                                                                                                                                                                                                          |
```

Comment:
I feel we may not need the column `Topic` since the names of GA already show the meaning.
Maybe we can add a new optional column `Desc` after `Status` to describe some additional information if needed.

Reply:
I've added the `Desc` column, and kept it empty for the time being. Or should we add this column later when it's actually needed?

Reply:
yeah let's do